projects/ngx-datatable/src/lib/components/body/body.component.ts

```diff
@@ -135,6 +138,7 @@ export class DataTableBodyComponent implements OnInit, OnDestroy {
   @Input() scrollbarV: boolean;
   @Input() scrollbarH: boolean;
   @Input() loadingIndicator: boolean;
+  @Input() ghostLoader: boolean;
```

Comment:
Thinking about the naming here: would it make sense to align the naming closer to the existing `loadingIndicator` one, e.g:

```suggestion
  @Input() ghostLoadingIndicator: boolean;
```

On a second note, it seems that the term `skeleton` load is more widely used as opposed to `ghost` loader. Would it make sense to all align it to `skeleton`, e.g:

```suggestion
  @Input() skeletonLoadingIndicator: boolean;
```

@chintankavathia @timowolf WDYT?